package.json

```diff
@@ -64,7 +64,6 @@
         "deps:fonts:roboto": "cpy node_modules/roboto-fontface/fonts/roboto-slab/Roboto-Slab-*.woff src/lib/roboto/fonts/",
         "predeps:tests": "npm run clean:deps:tests",
         "deps:tests": "run-p deps:tests:*",
-        "deps:tests:mockjax": "cpy node_modules/jquery-mockjax/dist/jquery.mockjax.js tests/lib/mockjax/js/",
```

Comment:
It looks like you've removed mockjax. Could you please update the Release notes, gitignore, and any other location that may still mention it?